framework/mesh/unpartitioned_mesh/unpartitioned_mesh.h

```diff
@@ -64,11 +64,14 @@ class UnpartitionedMesh
   const BoundBox& BoundingBox() const { return bound_box_; }
   void ComputeBoundingBox();
 
-  Options& MeshOptions() { return mesh_options_; }
+  void SetMeshOptions(const Options& opts) { mesh_options_ = opts; }
   const Options& MeshOptions() const { return mesh_options_; }
 
-  MeshAttributes& Attributes() { return attributes_; }
-  const MeshAttributes& Attributes() const { return attributes_; }
+  void SetType(MeshType type) { mesh_type_ = type; }
+  const MeshType& Type() const { return mesh_type_; }
+
+  void SetExtruded(bool extruded = true) { extruded_ = extruded; }
```

Comment:
While it makes sense to have this default to `true`, this routine appearing without an argument looks a little funny. Would it make more sense to just not have a default argument?